packages/services-store/src/types/translateAPI.ts

```diff
@@ -85,13 +85,52 @@ export interface MastheadLink {
 
 /**
  * A menu section for masthead
+ * @deprecated
  */
-export interface MastheadL1 {
+export interface LegacyMastheadL1 {
   title: string;
   url?: string;
   menuItems?: MastheadLink[];
 }
 
+/**
+ * A menu section for masthead
+ */
+export interface MastheadL1 {
+  title: string;
+  url?: string;
+  menuItems: L1MenuItem[];
+  actions?: {
+    cta?: BasicLink;
+    login?: BasicLink;
+  };
+}
+
+export interface L1MenuItem extends BasicLink {
+  submenu?: L1Submenu;
+}
+
+export interface L1Submenu {
+  announcement?: string; // From AEM rich text editor
+  menuSections: L1SubmenuSection[]; // maximum of 3 in outer array | renamed to be agnostic to the design
```

Comment:
I wonder if the maximum of three limitation should be enforced in code or part of content guidance. Either way, we can remove my part of this comment.

```suggestion
  menuSections: L1SubmenuSection[]; // maximum of 3 in outer array
```